docs/the-git-based-process-you-need-for-mainframe-development-DRAFT/pipeline-design-and-implementation-supporting-the-workflows.md

```diff
@@ -55,7 +55,7 @@ In this section, the technical implementation of the different composition of pi
 
 ![](images/media/image16.png) When the developers start working on a new task, they will first create a feature branch. Feature branches are created off the latest code state of the state of the source configuration -- the *main*, *epic* or *release* maintenance branch.
 
-If the feature branch was created on the central Git repository, the developers can use the IDE, a terminal, or another Git interface on their local workstation to clone or fetch the new feature branch from the central Git repository down to their local working tree in which the changes will be implemented.
+If the feature branch was created on the central Git repository, the developers can use the IDE, a terminal, or another Git interface on their local workstation to clone or pull the new feature branch from the central Git repository. They checkout the feature branch inw which the changes will be implemented.
```

Comment:
> They checkout the feature branch inw which the changes will be implemented.

Revise to:
"They check out the feature branch in which the changes will be implemented."

Reply:
I agree with removing the extraneous 'w', but I think 'checkout' is preferable since that's the git action on the cli. Also, `git switch` was introduced as a simpler alternative to the 'multitool' nature of `git checkout`.

Reply:
As this is a core git concept, I looked at https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging and they take the approach not to use *checkout* except as the command and use *switch* as the natural word in their narrative.

For example - https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell#_switching_branches

Perhaps that's the better approach.

Reply:
Ideally we should be relying on a basic level of git understanding here and not setting ourselves the additional challenge to teach the basics at the same time.

Reply:
So returning to the original comment, how about:

> "They switch to the feature branch in which the changes will be implemented."

Reply:
@ijmitch @dennis-behm I really like Ian's idea to just say "switch" branches, actually, as this can cover both Git commands (on a conceptual level), and the reader can just use whichever command they prefer to perform the action.

Reply:
I will even update my other review comment to reflect this, as well. :smile:

It's also much less clunky than having to say "check out (or checkout...) or switch to" every time.